cpp/src/cider-velox/benchmark/tpch/TpchInMemBuilder.cpp

```diff
@@ -151,7 +151,8 @@ TpchPlan TpchInMemBuilder::getQ1Plan(double scaleFactor) const {
           //   .tableScan(kLineitem, selectedRowType, fileColumnNames, {filter})
           .tableScan(Table::TBL_LINEITEM, std::move(selectedColumns), scaleFactor)
           .capturePlanNodeId(lineitemPlanNodeId)
-          .filter(filter)
+          // velox filter will wrap result childrens in dictionary
+          //   .filter(filter)
```

Comment:
Comment filters for debugging? Better to revert it if so.

Reply:
copy elimination doesn't support filter. So I commend all filter operations and just benchmark project op.